packages/tooltip/src/use-tooltip.ts

```diff
@@ -26,11 +26,11 @@ export interface UseTooltipProps {
   /**
    * Callback to run when the tooltip shows
    */
-  onShow?(): void
+  onOpen?(): void
```

Comment:
Our previous version has `onOpen` and `onClose` instead of `onShow` and `onHide`.

Throughout the package I can see both show & hide and open and close, I think it's better we have a unified naming here to clear the confusion

Reply:
I like this change because it also unifies `useTooltip` with an api similar to `useDisclosure`. @segunadebayo what do you think?

Reply:
This change is good. I definitely want us to keep same API as the previous version.

Thank you.

Reply:
Done. I have made these changes.